addOns/fuzz/src/main/java/org/zaproxy/zap/extension/fuzz/payloads/generator/NumberPayloadGenerator.java

```diff
@@ -48,9 +48,7 @@ public NumberPayloadGenerator(int fromNo, int toNo, int steps) {
 
     @Override
     public long getNumberOfPayloads() {
-        if (LOGGER.isDebugEnabled()) {
-            LOGGER.debug("Number of payloads = {}", (toNo - fromNo) / steps);
-        }
+        LOGGER.debug("Number of payloads = {}", (toNo - fromNo) / steps);
```

Comment:
It had the check to avoid doing the calculation twice, if you want to keep this way extract a variable and use it in both places.